IceCubesApp/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -55,14 +55,14 @@
 "settings.display.navigation-title" = "Anzeigeeinstellungen";
 "settings.display.restore" = "Standard wiederherstellen";
 "settings.display.section.display" = "Anzeigen";
-"settings.display.section.theme" = "Thema";
+"settings.display.section.theme" = "Design";
```

Comment:
While "Thema" is already a german word, I checked what other apps do (e.g. Notion, Things) and they often use "Design" which is acceptable in german and IMO fits better